JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.ts

```diff
@@ -0,0 +1,241 @@
+import { NgModule, Component, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import {
+  DxToolbarModule,
+  DxSelectBoxModule,
+  DxTemplateModule,
+  DxResizableModule,
+  DxDropDownButtonModule,
+  DxButtonGroupModule,
+  DxCheckBoxModule,
+  DxRadioGroupModule,
+} from 'devextreme-angular';
+
+import DataSource from 'devextreme/data/data_source';
+import notify from 'devextreme/ui/notify';
+import {
+  FontFamilies,
+  FontSize,
+  FontStyle,
+  LineHeight,
+  ListType,
+  Service,
+  TextAlign,
+  TextAlignExtended,
+} from './app.service';
+
+if (!/localhost/.test(document.location.host)) {
+  enableProdMode();
+}
+
+@Component({
+  selector: 'demo-app',
+  templateUrl: 'app/app.component.html',
+  styleUrls: ['app/app.component.css'],
+  providers: [Service],
+})
+
+export class AppComponent {
+  toolbarLineModes = [
+    {
+      text: 'Multiline mode',
+      value: true,
+    },
+    {
+      text: 'Singleline mode',
+      value: false,
+    },
+  ];
+
+  multiline: boolean;
+
+  lineHeight: number[];
+
+  fontSizes: FontSize[];
+
+  lineHeights: LineHeight[];
+
+  fontFamilies: FontFamilies[];
+
+  fontStyles: FontStyle[];
+
+  textAlignItems: TextAlign[];
+
+  textAlignItemsExtended: TextAlignExtended[];
+
+  selectedTextAlign: string[];
+
+  listTypes: ListType[];
+
+  undoButtonOptions: any;
+
+  redoButtonOptions: any;
+
+  fontSizeOptions: any;
+
+  lineHeightOptions: any;
+
+  fontFamilyOptions: any;
+
+  fontStyleOptions: any;
+
+  listOptions: any;
+
+  dateBoxOptions: any;
+
+  checkBoxOptions: any;
+
+  attachButtonOptions: any;
+
+  addButtonOptions: any;
+
+  removeButtonOptions: any;
+
+  aboutButtonOptions: any;
+
+  constructor(service: Service) {
+    this.multiline = true;
+    this.fontSizes = service.getFontSizes();
```

Comment:
Maybe we can initialize component fields in-line way?
It may look like this:


```
  fontSizes = this.service.getFontSizes();
  // other fields
  undoButtonOptions = {
    icon: 'undo',
    onClick: () => {
      notify('Undo button has been clicked!');
    },
  }
  // other component fields
  
  constructor(private service: Service) {}
```

If we do it, we can drop the 'any' types :)

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/3dd5d635fce75dad747b4036cbac4ef61f883e96